src/components/Input/Input.tsx

```diff
@@ -79,11 +84,14 @@ export const Input = React.forwardRef<HTMLInputElement, Props>(
                     aria-invalid={context === 'critical' ? 'true' : undefined}
                     data-lpignore={isLastPassDisabled}
                 />
+
                 {context === 'critical' && errorMessage ? (
                     <div {...elem('errorMessageWrapper')}>
                         <Error viewBox="0 0 24 24" {...elem('icon')} />
                         <p {...elem('errorMessage', { context })}>{errorMessage}</p>
                     </div>
+                ) : reserveErrorMessageSpace && !helperText ? (
+                    <div {...elem('errorMessageWrapper', { reserveErrorMessageSpace })} />
```

Comment:
I'm surprised eslint doesn't complain about nested tenantry... but I wonder if we can simplify it anyway. like... maybe just in the CSS. render the `errorMessageWrapper` when `context === 'critical' && errorMessage ||  reserveErrorMessageSpace && !helperText`. Then pass the `reserveSpace` prop to the `elem` and have that `min-height` added there. That would also mean that the same div is used both to reserve the space, and to render the error. Do you think that would work?

Reply:
It would not work because, in this case, the error message will always be displayed when validation is intended, as the error message will always be provided. Therefore, the empty div should be displayed separately. Nested ternaries are not ideal, but I find them more understandable than the other options I've tried 😅. I've pushed another possible option to avoid using nested ternaries. Please have a look. :-)

Reply:
or I can control visibility of error message with css, it looks more understandable, but I am not a big fun of it. What do you think?
```typescript
          {(context === 'critical' && errorMessage) ||
                (reserveErrorMessageSpace && !helperText) ? (
                    <div {...elem('errorMessageWrapper')}>
                        <Error
                            viewBox="0 0 24 24"
                            {...elem('icon', { context, reserveErrorMessageSpace })}
                        />
                        <p
                            {...elem('errorMessage', {
                                context,
                                reserveErrorMessageSpace,
                            })}
                        >
                            {errorMessage}
                        </p>
                    </div>
                ) : (
                    helperText && <p {...elem('helperText')}>{helperText}</p>
                )}
```

Reply:
I like the last one more :D

Reply:
I thought with the fancy TS you made, that error message wouldn't be provided unless there is an error... I think our form library would have no error messages otherwise. also one field in a form can have different validation rules, and accordingly different error messages. E.g. `field is required` or `please provide a valid email address`.
but indeed, if this is not the case, we need to test on every part of the code.